test/Dialect/FIRRTL/lower-layers.mlir

```diff
@@ -763,3 +763,24 @@ firrtl.circuit "RWTH" {
     }
   }
 }
+
+// -----
+
+// Check sv.verbatim inner refs are updated, as occurs with views under layers.
+// CHECK-LABEL: circuit "Verbatim"
+firrtl.circuit "Verbatim" {
+  firrtl.layer @ViewLayer  bind { }
+  firrtl.module @Verbatim() {
+    firrtl.layerblock @ViewLayer {
+      %c1_ui10 = firrtl.constant 1 : !firrtl.uint<10>
+      %n = firrtl.node sym @node %c1_ui10 : !firrtl.uint<10>
+      sv.verbatim "// node: {{0}}, {{1}}"(%n) : !firrtl.uint<10> {symbols = [#hw.innerNameRef<@Verbatim::@node>]}
+    }
+  }
+}
+// CHECK:        firrtl.module private @Verbatim_ViewLayer() {
+// CHECK-NEXT:     firrtl.constant 1
+// CHECK-NEXT:     %[[N:.+]] = firrtl.node sym @[[SYM:.+]] %
+// CHECK-NEXT:     sv.verbatim 
+// CHECK-SAME:     !firrtl.uint<10> {symbols = [#hw.innerNameRef<@Verbatim_ViewLayer::@[[SYM]]>]}
+// CHECK-NEXT:   }
```

Comment:
Nit: using a substitution for `@node` seems incorrect as this inner symbol should be preserved? Conversely, _not_ using a substitution block for the resulting `@Verbatim_ViewLayer` seems wrong. I'd invert these to not use a substitution for the former and use a substitution for the latter.

Reply:
Good call, thanks!  I wasn't sure if it was /necessary/ the inner symbol was re-used as much as a happenstance of implementation for simplicity/efficiency.

Inverting sounds good to me, will update!